cmake/std/atdm/README.md

```diff
@@ -534,8 +558,8 @@ contents:
     environment.sh  # Load env for the given system based on $JOB_NAME keys
     all_supported_builds.sh  # [Optional] List of all supported builds
     tweaks/
-       <COMPILER0>-<BUILD_TYPE0>-<NODE_TYPE0>.cmake  # [Optional]
-       <COMPILER1>-<BUILD_TYPE1>-<NODE_TYPE1>.cmake  # [Optional]
+       <COMPILER0>-<BUILD_TYPE0>-<NODE_TYPE0>-<KOKKOS_ARCH0>.cmake  # [Optional]
+       <COMPILER1>-<BUILD_TYPE1>-<NODE_TYPE1>-<KOKKOS_ARCH0>.cmake  # [Optional]
```

Comment:
How come this isn't `KOKKOS_ARCH1`?

Reply:
I think this was a typo on my part but this is actually more accurate.  For example, these could be:

```
GNU-DEBUG-SERIAL-HSW.cmake
INTEL-RELEASE-OPENMP-HSW.cmake
```

Hopefully people get the idea anyway.

Reply:
Thanks Ross!